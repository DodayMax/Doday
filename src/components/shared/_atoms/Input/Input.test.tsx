import * as React from 'react';
import { shallow, mount } from 'enzyme';
import { Input } from '@components';

it('renders correctly without props', () => {
  const input = shallow(<Input />);

  expect(input).toMatchSnapshot();
});