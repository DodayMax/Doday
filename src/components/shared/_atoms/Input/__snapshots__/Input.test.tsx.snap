// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly without props 1`] = `ShallowWrapper {}`;
